src/libs/PusherUtils.ts

```diff
@@ -1,26 +1,25 @@
+import {OnyxUpdate} from 'react-native-onyx';
 import CONFIG from '../CONFIG';
 import Log from './Log';
 import NetworkConnection from './NetworkConnection';
 import * as Pusher from './Pusher/pusher';
 import CONST from '../CONST';
+import {OnyxUpdateEvent, OnyxUpdatesFromServer} from '../types/onyx';
+
+type PushJSON = OnyxUpdateEvent[] | OnyxUpdatesFromServer;
+
+type Callback = (data: OnyxUpdate[]) => Promise<void>;
+
+type OnEvent = (pushJSON: PushJSON) => void;
```

Comment:
I don't think we have to declare a separate type for this

Reply:
You can just specify the type inline here

```ts
function subscribeToPrivateUserChannelEvent(eventName: string, accountID: string, onEvent: OnEvent)
```

Reply:
@hayata-suenaga I've updated it